x-pack/solutions/security/packages/expandable-flyout/src/components/preview_section.tsx

```diff
@@ -134,21 +135,16 @@ export const PreviewSection: React.FC<PreviewSectionProps> = memo(
       <div
         css={css`
           position: absolute;
-          top: 8px;
-          bottom: 8px;
-          right: 4px;
+          top: 0px;
+          bottom: 0px;
+          right: 0px;
           width: ${width};
           z-index: 1000;
+          padding: ${euiTheme.size.m} ${euiTheme.size.s} 0px ${euiTheme.size.s};
+          background: ${transparentize(euiTheme.colors.shadow, 0.1)};
```

Comment:
in theory we're not supposed to use `transparentize` anymore (see https://github.com/elastic/kibana/issues/201888). Though I don't think we have an alternative at the moment... Maybe check with UI, but if no alternative, let's add a comment: // TODO EUI: add color with transparency. And if you don't mind adding he same comment [here](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/timelines/components/modal/index.styles.tsx#L30) as well?  I was on a zoom with EUI and they asked me to add the comment there, I just didn't want to create a PR just for that one comment...